example/routing_asgi.py

```diff
@@ -0,0 +1,43 @@
+from typing import cast
+
+from haberdasher_connecpy import HaberdasherASGIApplication
+from service import HaberdasherService
+from starlette.applications import Starlette
+from starlette.responses import PlainTextResponse
+from starlette.routing import Mount, Route
+from starlette.types import ASGIApp
+
+haberdasher_app = HaberdasherASGIApplication(HaberdasherService())
+
+
+def strip_prefix(prefix: str, app: ASGIApp) -> ASGIApp:
+    async def stripped_app(scope, receive, send):
+        scope["path"] = scope["path"].removeprefix(prefix)
+        return await app(scope, receive, send)
+
+    return stripped_app
+
+
+app = Starlette(
+    routes=[
+        Route(
+            "/healthz",
+            lambda _: PlainTextResponse("OK"),
+        ),
+        Mount(
+            "/services",
+            app=cast(ASGIApp, haberdasher_app),
+        ),
+        Mount(
+            "/moreservices",
+            app=Mount(
+                haberdasher_app.path,
+                app=strip_prefix("/moreservices", cast(ASGIApp, haberdasher_app)),
```

Comment:
Only multiple services under a service prefix is more tedious, actually I was somewhat surprised starlette doesn't provide strip functionality out of the box like Go does. But it's such a small use case, and the strip middleware is very small, so IMO it's ok to avoid implementing routing in connecpy just to support it. What do you think?